api/src/main.py

```diff
@@ -1,19 +1,77 @@
+import sys
+from contextlib import asynccontextmanager
+
 from fastapi import FastAPI
-from sqlmodel import SQLModel
+from sqlalchemy.exc import SQLAlchemyError
+from sqlmodel import Session, SQLModel, select
 from src.db import engine
 from uvicorn import run
 
-from .routers import company, user
+from .routers import achievement, company, shop, user
 
 SQLModel.metadata.create_all(bind=engine)
 
-app = FastAPI()
 
+def _populate_achievements() -> None:
+    """Populate the Achievements table."""
+    from .models import Achievement as AchievementModel
+
+    reach_level_5: AchievementModel = AchievementModel(
+        name="Reach level 5.", description="Achievement earned when reaching Level 5."
+    )
+    reach_level_10: AchievementModel = AchievementModel(
+        name="Reach level 10.", description="Achievement earned when reaching Level 10."
+    )
+    reach_level_20: AchievementModel = AchievementModel(
+        name="Reach level 20.", description="Achievement earned when reaching Level 20."
+    )
+    reach_level_30: AchievementModel = AchievementModel(
+        name="Reach level 30.", description="Achievement earned when reaching Level 30."
+    )
+    reach_leaderboard: AchievementModel = AchievementModel(
+        name="Leaderboard Achieved!", description="Achievement earned when finding your way onto the Leaderboard!"
```

Comment:
This should probably be reworded
```suggestion
        name="Leaderboard Achieved!", description="Achievement earned after finding your way onto the Leaderboard!"
```